web/src/components/pages/dashboard/dashboard.css

```diff
@@ -71,6 +76,7 @@
 
         a {
             margin-inline-end: 25px;
+            overflow-wrap: anywhere;
```

Comment:
Hmm, this starts wrapping even when all three words are still visible on the page. I personally don't think it's necessary - it's not ideal for the last word to be cut off (if it is, depending on the language) but with individual letters wrapped it seems even harder to read.

Reply:
fixed :)